exasol_sagemaker_extension/autopilot_handler.py

```diff
@@ -1,4 +1,5 @@
 import os.path
+from sagemaker import Predictor
```

Comment:
I think, we should split this file, because now it contains training, endpoint handling, .... Per use case one file is probably better.

Reply:
as you said it violate SRP. But I have designed it as a collection of static methods which uses AWS APIs. I might divide them into non-static classes?